pkgs/tools/video/yamdi/default.nix

```diff
@@ -0,0 +1,36 @@
+{
+  stdenv,
+  fetchurl,
+  ...
+}:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  name = "yamdi-${version}";
+  version = "1.9";
+
+  # Source repo is also available here:
+  # https://github.com/ioppermann/yamdi
+  src = fetchurl {
+    url = "https://downloads.sourceforge.net/project/yamdi/yamdi/${version}/yamdi-${version}.tar.gz";
+    sha256 = "4a6630f27f6c22bcd95982bf3357747d19f40bd98297a569e9c77468b756f715";
+  };
+
+  buildFlags = "CC=${stdenv.cc}/bin/cc";
+
+  installPhase = ''
+    mkdir -p $out/bin
+    install -m 0755 yamdi $out/bin/yamdi
```

Comment:
`install -D yamdi $out/bin` will create all directories as well. `mkdir` can then be omitted. Setting file mods is unnecessary since nix reset them anyway.